.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+	"useTabs": true,
+	"semi": true,
+	"singleQuote": false,
```

Comment:
singleQuote being true or false is always debatable, but let's stick to [airbnb](https://github.com/airbnb/javascript#strings--quotes) style guides which uses single quote